Makefile

```diff
@@ -29,9 +29,12 @@ build: ## Build/compile the OpenFGA service
 run: build ## Run the OpenFGA server with in-memory storage
 	./bin/openfga run
 
+.PHONY: migrate-postgres
+migrate-postgres: build
```

Comment:
shouldn't we also offer `migrate-down` in case they want to rollback?

Reply:
Added a `--version` flag that you can set to `0` (or any version less than the current one) that will roll you back.